### PR TITLE
Sep6 66 login function

### PR DESCRIPTION
**Pull request by HolaSombreros**
Azure Function for Login

## Description
Added a repository for User, Azure Http Trigger function for login and add custom messages on exceptions

## Related Issue(s)

https://via-group2.atlassian.net/browse/SEP6-66

## How Has This Been Tested?

Tested locally with postman

### CODE REVIEW

File: src/MovieManagement.Database/Config/DatabaseConfig.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace MovieManagement.Database.Config;
```
Comment (stefaniatomuta) [inline src/MovieManagement.Database/Config/DatabaseConfig.cs:1]: You have conflicts to solve before merge with main

File: src/MovieManagement.Database/Entities/UserEntity.cs [hunk 2]
```diff
@@ -5,11 +5,11 @@ public class UserEntity
 {
     public Guid UserId { get; set; }
 
-    public string Username { get; set; }
+    public string Username { get; set; } = default!;
+    
+    public string Email { get; set; } = default!;
```
Comment (stefaniatomuta) [inline src/MovieManagement.Database/Entities/UserEntity.cs:10]: Do you have validators for email? Maybe we should consider Fluent Validation

**Review by stefaniatomuta — CHANGES_REQUESTED** [review]

File: src/MovieManagement.Database/Entities/UserEntity.cs (hunk 2 above)
Comment (HolaSombreros) [reply src/MovieManagement.Database/Entities/UserEntity.cs:10]: All the other models and dtos have the [Email] constraint so technically speaking there is validation before it reaches the database. Do you think it is needed here as well?

Comment (stefaniatomuta) [reply src/MovieManagement.Database/Entities/UserEntity.cs:10]: I am not sure whether the [Emai] is working, but maybe it is

**Review by stefaniatomuta — APPROVED** [review]

**Pull request merged**